File: week2/day4/TheGardenApplication/TheGardenApplication/Flower.cs
using System;
using System.Collections.Generic;
using System.Dynamic;
using System.Text;

namespace TheGardenApplication
{
    class Flower
    {
        public double WaterAmount { get; set; }
        public string Color { get; set; }

        public Flower(string color)
        {
            this.Color = color;
        }
        public void watering(double water)
        {
            this.WaterAmount += (water * 0.75);
        }

        public bool IfNeedWater()
        {
            if (this.WaterAmount < 5)
            {
                return true;
            }
            else
            {
                return false;
            }
        }

    }
}

File: DSA-2019/week-02/day-1/flower.py
class Flower():
    def __init__(self, color, water_amount=0):
        self.water_amount = water_amount
        self.color = color

    def need_water(self):
        return self.water_amount < 5

    def absorb_water(self, water):
        self.water_amount += water * 0.75
